src/library/scala/collection/mutable/HashTable.scala

```diff
@@ -39,7 +39,7 @@ import java.lang.Integer
  *
  *  @tparam A     type of the elements contained in this hash table.
  */
-private[mutable] abstract class HashTable[A, B, Entry >: Null <: HashEntry[A, Entry]] extends HashTable.HashUtils[A] {
+private[collection] /*abstract class*/ trait HashTable[A, B, Entry >: Null <: HashEntry[A, Entry]] extends HashTable.HashUtils[A] {
```

Comment:
Why leave the commented abstract class here?

Reply:
To remind me that I should try to go back to an abstract class, if possible.

Reply:
could you add a comment to that effect?

Reply:
I'm confused, is this planned to be reverted to an abstract class at some time in the future?